templates/zerver/portico_error_pages/demo_creation_disabled.html

```diff
@@ -0,0 +1,20 @@
+{% extends "zerver/portico_error_pages/portico_error_page.html" %}
+
+{% block title %}
+<title>{{ _("Demo organizations disabled") }} | Zulip</title>
+{% endblock %}
+
+{#
+Content below is not marked for translation intentionally, as most
+users will not encounter this page.
+#}
```

Comment:
I would put this in the `errorcontent` right before the two strings; I think otherwise, some folks will just fail to read this.

Reply:
Moved comment to be inside the `errorcontent` div in updated changes.